src/reference/00-Getting-Started/05-Basic-Def.md

```diff
@@ -138,8 +97,8 @@ the value `"hello"`.
 If you use the wrong value type, the build definition will not compile:
 
 ```scala
-lazy val root = (project in file(".")).
-  settings(
+lazy val root = (project in file("."))
+  .settings(
     name := 42  // will not compile
   )
 ```
```

Comment:
No it's no longer true since 0.13.7, so we can replace it with a warning for ppl to upgrade to a recent version at the top of the page.